src/Modules/OrchardCore.Commerce/Views/TextProductAttributeField.cshtml

```diff
@@ -6,6 +6,12 @@
     var settings = attr?.Settings as TextProductAttributeFieldSettings;
     var id = $"{Model.IdPrefix}-attribute-{index}";
     var optionIndex = 0;
+
+    var sku = (string)Model.Sku;
+    var canBeBoughtDictionary = ((Dictionary<string, bool>)Model.CanBeBought);
+    var availableVariants = canBeBoughtDictionary.Where(keyValuePair => keyValuePair.Value == true);
```

Comment:
`canBeBoughtDictionary` might return null thus a null check would be welcome here for safety reasons.